_rules/presentational-children-no-focusable-content-307n5z.md

```diff
@@ -0,0 +1,144 @@
+---
+id: 307n5z
+name: Element with presentational children has no focusable content
+rule_type: atomic
+description: |
+  This rule checks that elements with a role that makes its children presentational do not contain focusable elements.
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag20:4.1.2: # Name, Role, Value (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any HTML or SVG element with a [semantic role][] that defines its [children][child] to be [presentational children][].
+
+## Expectation
+
+None of the target elements have [descendants][] in the [flat tree][] that are part of [sequential focus navigation][].
+
+## Assumptions
+
+This rule assumes that elements that are part of [sequential focus navigation][] do not immediately blur, or move focus to another element. Such elements will fail this rule, but may still satisfy success criteria 1.3.1 and 4.1.2.
```

Comment:
As per my previous comment about Focus order. If true, the criterion should be added.
```suggestion
This rule assumes that elements that are part of [sequential focus navigation][] do not immediately blur, or move focus to another element. Such elements will fail this rule, but may still satisfy success criteria 1.3.1, 2.4.3, and 4.1.2.
```